spec/dummy/bin/prepare-environment

```diff
@@ -9,7 +9,9 @@ pushd $(dirname $(dirname $(dirname ${__dir})))
 popd
 
 pushd $(dirname ${__dir})
+    bundle exec rake assets:precompile
```

Comment:
@mkasztelnik I guees that if there is no ``config.assets.compile = false`` in ``spec/dummy/config/environments/development.rb`` like in ``spec/dummy/config/environments/production.rb`` then after making some commands using ``bundle exec rake`` will automagic recompile assets by default?